CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This release brings the following significant changes:
  - Upgrade to Cassandra driver 4
  - Migrate to OpenSearch
  - Review our threading model to cap threads performing blocking tasks
+ - Deprecating custom JMAP quotas specification 
  - Implement official JMAP quotas specification
 
```

Comment:
```suggestion
 - Upgrade to Cassandra driver 4
 - Migrate to OpenSearch
 - Review our threading model to cap threads performing blocking tasks
 - Implement official JMAP quotas specification

### Deprecations

 - Deprecating custom JMAP quotas specification 
```

Rationals: this is not a significant change to me.

Reply:
not used to see a section for that sorry. It's fine, it's quick and makes relatively sense :)